src/views/homepage/components/merchandising-slots/slots/hashiconf-global-slot/index.tsx

```diff
@@ -2,9 +2,10 @@ import Image from 'next/image'
 import VisuallyHidden from '@reach/visually-hidden'
 import CardLink from 'components/card-link'
 import Text from 'components/text'
+import { HashiConfGlobalSlotProps } from './types'
 import s from './hashiconf-global-slot.module.css'
 
-function HashiConfGlobalSlot() {
+function HashiConfGlobalSlot({ description }: HashiConfGlobalSlotProps) {
```

Comment:
For merchandising slots, it didn't feel feasible to expose much content, as it feels pretty tightly coupled to the presentation here (which I think is kind of a good thing on more "polished" pages - trades off some content flexibility for highly intentional polish 🙌 )

Reply:
Yeah, these merch slots are pretty bespoke, and still unsure there is any pattern to follow that would allow JSON content to easily modify these that wouldn't be too overwhelming.